Assets/Scripts/Models/Buildable/Furniture.cs

```diff
@@ -772,6 +821,14 @@ public void Deconstruct()
             furniture.Jobs.CancelAll();
         }
 
+        if (deconstructInventory != null)
+        {
+            foreach (Inventory inv in deconstructInventory)
+            {
+                World.Current.inventoryManager.PlaceInventory(Tile, inv);
```

Comment:
I'm not up to date with the latest inventory stuff, will placing multiple inventory on the same tile work?